app/translations/en/validation.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  validation:
+    blank: This field is required
+    invalid_email_or_password: Invalid email or password
+    email:
+      required: Please provide your email address
```

Comment:
for email it will be even worse - if some modules might depend on the core module translation, you changed [en.validation.email from string](https://github.com/Platform-OS/pos-module-core/blob/master/modules/core/public/translations/en/validation.yml#L7) to hash, which can cause 500 - probably not the case currently, but something to consider

Reply:
It was based on https://github.com/Platform-OS/pos-module-user/blob/master/modules/user/public/views/partials/lib/commands/user/create/check.liquid#L5 but it seems I don't even need this any more on the app level.
It shouldn't cause any issues with the core module as it uses its own prefixed key `modules/core/validation.email` https://github.com/Platform-OS/pos-module-core/blob/master/modules/core/public/views/partials/lib/validations/email.liquid#L11 👍